src/components/NavBar/ShoppingBag.css.ts

```diff
@@ -0,0 +1,22 @@
+import { style } from '@vanilla-extract/css'
+import { sprinkles } from 'nft/css/sprinkles.css'
+
+export const bagQuantity = style([
+  sprinkles({
+    position: 'absolute',
+    top: '4',
+    right: '4',
+    backgroundColor: 'blue400',
```

Comment:
is this blue for dark and lightmode or do we use pink in lightmode?

Reply:
will check in with design